Automattic/simplenote-electron
# Remove sort options bar and update display settings

### Fix

Fixes #2769 

This removes the sort options bar from the note list and updates the sort options in the Display Settings to match the options which the sort bar offered. 

**Screenshots**

<details>
<summary>Before</summary>
<img width="333" alt="Screen Shot 2021-04-13 at 8 40 34 AM" src="https://user-images.githubusercontent.com/1326294/114547049-4da7b100-9c34-11eb-8f06-5cb4368a2e2a.png">
<img width="630" alt="Screen Shot 2021-04-13 at 8 41 07 AM" src="https://user-images.githubusercontent.com/1326294/114547057-513b3800-9c34-11eb-87f0-6febc638c266.png">
</details>
<details>
<summary>After</summary>
<img width="336" alt="Screen Shot 2021-04-13 at 8 41 47 AM" src="https://user-images.githubusercontent.com/1326294/114547066-55675580-9c34-11eb-8655-8afd69bf4a89.png">
<img width="621" alt="Screen Shot 2021-04-13 at 8 42 14 AM" src="https://user-images.githubusercontent.com/1326294/114547079-59937300-9c34-11eb-86b6-8b906631cb75.png">
</details>

### Test

1. Ensure sort options do not show in the note list.
2. Go to display settings.
3. Ensure the sort options show completely.
4. Ensure the sort options sort appropriately.

### Release

- Removed the sort options bar from the note list and updated the sort options in the Settings.
